### ISSUE TITLE
[Suggestion] Consecutive commands

### ISSUE THREAD

**Issue by cyruscook** [cabaletta/baritone#448]
Little suggestion, it would be great if we could put multiple commands in at once that would run one by one, for example
`goto -2012 80 8212 then disconnect`
`goal 0 0 then invert`
`goto portal then goto 7213 8212` (Go to nether coords)
`build portal then goto 7213 8212`
`goto 2312 721 then build bed then use bed then disconnect` (Go to a certain location then set your spawn there and then disconnect)

You can see that I've also suggested some new commands to compliment this, such as `use`.

**Comment by 2b2tQueueProject** [comment]
so... scripting?

I was thinking of doing something like this but gave up when I was trying to make for loops, while loops, and if statements.

btw this should be a baritone issue

**Comment by cyruscook** [comment]
Yeah scripting kinda. Loops and if statements are probably overkill but just simple thens.

It is a baritone issue?

**Comment by 2b2tQueueProject** [comment]
lol im a retard lul. I thought this was the impact repo.

**Comment by SylveonBottle** [comment]
Duplicate of #269, [according to l1ving](https://github.com/cabaletta/baritone/issues/269#issuecomment-751557525)

**Comment by CesiumCs** [comment]
you have helped nobody, thanks!
